common/src/main/java/org/apache/rocketmq/common/namesrv/NamesrvConfig.java

```diff
@@ -78,6 +78,9 @@ public class NamesrvConfig {
      */
     private boolean enableControllerInNamesrv = false;
 
+    private volatile boolean needWaitForService = true;
+
+    private int waitSecondsForService = 45;
```

Comment:
Hard-coded value is not optimal, users might adjust the interval of broker registration... Would it be better to set it to alpha * interval_of_broker_registration, where alpha is somewhat 1.5 or 2?

Reply:
broker config and namesrv config is independence，it's hard to get broker interval_of_broker_registration in namesrv.